content/kb/troubleshooting-failed-rwx-mount-due-to-connection-timeout.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Troubleshooting: Failed RWX mount due to connection timeout"
+author: Eric Weber
+draft: false
+date: 2023-09-05
+categories:
+  - "nfs"
+  - "rwx"
+  - "sles"
+  - "flannel"
+---
+
+## Applicable versions
+
+Confirmed with:
+
+- K3s `v1.24.8+k3s1` and RKE2 `v1.24.8+rke2r1`
+- Longhorn `v1.3.3`
```

Comment:
Should be applied to any Longhorn versions, right? Since this is not a version-specific issue.

Reply:
I mentioned this on line 24. I wanted to be specific on line 18 about which version we have definitely seen it on. Would it be better if I put the "Likely possible with:" block first for clarity?

Reply:
Oh I see, then its okay.